docs/docs-beta/docs/dagster-plus/saved-views.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Tailoring your search experience with catalog views"
+displayed_sidebar: "dagsterPlus"
+sidebar_label: "Customizing the catalog with views"
+---
+
+# Tailoring your search experience with catalog views
+
+Catalog views enable you to filter down your view of the Dagster Asset catalog in Dagster+. You can save catalog views for your own use, or share them with your team.
+
```

Comment:
Should we list out somewhere exactly which pages are affected by catalog views?

Catalog search page,
Asset catalog table,
Asset graph,
Asset health dashboard.

Reply:
I'm inclined to say no, since the UI makes it fairly obvious that there are filters applied. Curious what others think though

Reply:
It might be nice to know  --  I didn't actually know where it applied